server/test/services/export/JsonExporterTest.java

```diff
@@ -163,6 +181,313 @@ private void testApplicationTopLevelAnswers(
     resultAsserter.assertValueAtPath("$[" + resultIndex + "].submitter_email", "Applicant");
   }
 
+  // ADDRESS
```

Comment:
Nit: not a big fan of the question type delineators since they will almost definitely get out of sync with the tests themselves over time. The unit test names should be sufficient imo.

Reply:
done